packages/cli/commands/project/deploy.js

```diff
@@ -18,7 +18,7 @@ const { projectNamePrompt } = require('../../lib/prompts/projectNamePrompt');
 const { buildIdPrompt } = require('../../lib/prompts/buildIdPrompt');
 const { i18n } = require('../../lib/lang');
 const { uiBetaTag } = require('../../lib/ui');
-const { getAccountConfig } = require('@hubspot/cli-lib');
+const { getAccountConfig } = require('@hubspot/cli-lib/');
```

Comment:
Why does this need the trailing slash?

Reply:
I'm not sure how that got there. This should actually be a local-dev-lib import anyway